saskatoon/sitebase/static/css/bootstrap4.min.css

```diff
@@ -0,0 +1,7 @@
+/*!
+ * Bootstrap v4.0.0 (https://getbootstrap.com)
```

Comment:
i think we're already using bootstrap v3 somewhere, I' mo unsure if we should use both...

Reply:
yes the bootstrap v4 is just for the crispy form. the rest of the codebase still uses v3

Reply:
About the issue of conflicts, i think it is possible to work on it via the css tags used by bootstrap which are <b>,<strong>but manually adding a fix to these tags will affect other parts of the code base that use the tag. And trying to use classes would mean trying to edit it from the crispy end. i'll work on a quick fix and try to make it stable.

Reply:
So that's a good reason not wanting to add both bootstrap 3 and 4, I think.